bibliopixel/drivers/driver_base.py

```diff
@@ -2,12 +2,14 @@
 
 
 class ChannelOrder:
-    RGB = [0, 1, 2]
-    RBG = [0, 2, 1]
-    GRB = [1, 0, 2]
-    GBR = [1, 2, 0]
-    BRG = [2, 0, 1]
-    BGR = [2, 1, 0]
+    RGB = 0, 1, 2
```

Comment:
Wow... weird. NEVER seen that. Wait... that's making a tuple, not a list isn't it? Then yes I have. Duh.